.github/workflows/deployDemo.yml

```diff
@@ -45,7 +45,7 @@ jobs:
           curl -sL "https://keyserver.ubuntu.com/pks/lookup?op=get&search=0x2EE0EA64E40A89B84B2DF73499E82A75642AC823" | sudo apt-key add
           sudo apt-get update
           sudo apt-get install sbt -y
-          . ./sourceme.sh && optic_build
+          . ./sourceme.sh && optic_build_for_release
```

Comment:
kind of a tangent but `. ./sourceme.sh` is super confusing syntax imo. i know this is correct here, but its somewhat ambiguous because you arent sure if sourceme.sh is executable and maybe its a typo and should be `../sourceme.sh `, etc. `. sourceme.sh` or `source ./sourceme.sh` are significantly clearer imo

Reply:
You're totally right. I'll be a good scout and clean that up.